### PR TITLE
Add claim button

### PR DESCRIPTION
**Pull request by Joozty**

### CODE REVIEW

**Comment by Joozty** [comment]
@viktor-yakubiv This is just very basic PR, from the design perspective it's a disaster but I don't know how to make it look better. I am open to any suggestions. :)

**Review by viktor-yakubiv — COMMENTED** [review]
I propose to change buttons to
- **Request Dashboard** with the title _I am NAME’s manager and want to access CORE Dashboard_
- **Finish** with the title _I am not a manager of NAME_

The first button is shown for every case we have unclaimed repository either new or already existing. We need @oacore/api team support to expose if the repository was claimed already, if it's not there yet.

The second button should be ‘Finish’ for only new repositories but for existing ones it should not appear.

File: config.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ const validate = (config) =>
 const env = { local, development, production }
 const config = {
   ...env.production,
-  ...env[NODE_ENV],
+  // ...env[NODE_ENV],
```
Comment (viktor-yakubiv) [inline config.js:23]: Is this a mistake?

Comment (Joozty) [reply config.js:23]: yeah I was trying to avoid the bug discussed on slack

**Comment by viktor-yakubiv** [comment]
I cannot test it, is it okay at the moment?

File: pages/data-providers/index.jsx [hunk 2]
```diff
@@ -64,11 +64,17 @@ const getHelperMessage = ({ created, duplicated, error }) => {
...
-          able to approve it.
+          <p>
+            We found {created.name} under the entered address and added it to
+            our data provider collection. We will start harvesting as soon as we
+            are able to approve it.
+          </p>
+          <p>
+            After the approval we will sent a confirmation email to{' '}
+            {created.email}.
+          </p>
```
Comment (viktor-yakubiv) [inline pages/data-providers/index.jsx:77]: Sorry, I suggested this. I am thinking that this is semantically wrong. A paragraph cannot be descendant fo the paragraph, i.e. if we wrap the input into a paragraph, the only plain text should appear here. What do you think @Joozty?

```suggestion
          We found {created.name} under the entered address and added it to
          our data provider collection. As soon as we approve adding, we will
          start harvesting and sent a confirmation email to{' '}
          <Link
            href={`mailto:${created.email}`}
            title="the administrator email address"
          >
            {created.email}
          </Link>.
```

I hope, this is shorter.

File: store/data-provider.js [hunk 3]
```diff
@@ -45,6 +45,8 @@ class DataProvider {
     this.created = null
     this.duplicated = null
     this.error = null
+    this.isClaimingInProgress = false
+    this.claimData = {}
```
Comment (viktor-yakubiv) [inline store/data-provider.js:49]: Is this leftover?

File: templates/data-providers/styles.module.css [hunk 4]
```diff
@@ -15,3 +15,7 @@
     margin-left: 1rem;
   }
 }
+
+.form {
+  margin-top: 1rem;
```
Comment (viktor-yakubiv) [inline templates/data-providers/styles.module.css:20]: Is it a problem of the headline spacing?

**Review by viktor-yakubiv — APPROVED** [review]
Good job! 👍

File: templates/data-providers/styles.module.css (hunk 4 above)
Comment (Joozty) [reply templates/data-providers/styles.module.css:20]: yes, the margin is taken from [card title](https://github.com/oacore/design/blob/master/src/elements/card/card.css#L20) and it didn't look great so I extended it a little bit.

**Pull request merged**